### ISSUE TITLE
Unable to Post Replies

### ISSUE THREAD

**Issue by Freeesia** [drasticactions/FishyFlip#34]
I am developing an app that posts to Bluesky using this library.
To post a reply on Bluesky, it is necessary to fill in the values in the following fields:
https://github.com/bluesky-social/atproto/blob/main/lexicons/app/bsky/feed/post.json#L29
This exists in the library's data model as [Post.Reply](https://github.com/drasticactions/FishyFlip/blob/main/src/FishyFlip/Models/Post.cs#L77). 
However, I cannot pass a Reply when posting through `ATProtoRepo.CreatePostAsync()`. 
Could you please make a modification to allow passing a `Reply`?

I believe I can also help with the implementation.

Thank you.

**Comment by drasticactions** [comment]
Oops. Yeah, I missed it, fixed with https://github.com/drasticactions/FishyFlip/commit/04df284b302799d8bf1914bd8f88b12a70594481.

![image](https://github.com/drasticactions/FishyFlip/assets/898335/63c34f90-8c8b-4815-86b6-af9058175995)

![image](https://github.com/drasticactions/FishyFlip/assets/898335/7c65273c-9261-4dfb-9830-be3ce3a6829d)

I need to create docs for this still but to use it, you can look at the new test for it:

https://github.com/drasticactions/FishyFlip/commit/04df284b302799d8bf1914bd8f88b12a70594481

https://github.com/drasticactions/FishyFlip/blob/04df284b302799d8bf1914bd8f88b12a70594481/src/FishyFlip/Models/ReplyRef.cs#L10

https://github.com/drasticactions/FishyFlip/blob/04df284b302799d8bf1914bd8f88b12a70594481/src/FishyFlip/Models/Reply.cs#L10

In short, create a new `Reply` object, and pass in the `ReplyRef` root to the first post in the given thread and a `ReplyRef` parent to the post you're responding too.

https://www.nuget.org/packages/FishyFlip/1.7.43-alpha

It should be on Nuget now, let me know if this works for you. Thanks!

**Comment by Freeesia** [comment]
Thank you very much for the swift resolution and for addressing the issue promptly.
I truly appreciate your effort. Due to my current workload, it may take a few weeks before I can test the modification.I will certainly inform you about its performance as soon as possible.

Thanks once again!

**Comment by drasticactions** [comment]
I am closing this issue as I believe it has been addressed, if you're still having issues with it, feel free to reopen it. Thanks!
